File: Java/1.1 Classes and Objects.java
class Bank
{
    int x = 4;
    void show()
    { 
        System.out.println("Demo Function");
    }
}

public class Main
{
	public static void main(String[] args) {
		Bank ob = new Bank();
		ob.show();
		System.out.println(ob.x);
	}
}

File: Python/1.1 Classes and Objects.py
class Car:
    #Constructor
    def __init__(self,c,s):
        self.col = c 
        self.seats = s
    
    def show(self):
        print("Colour is:",self.col)
        print("Seat Count is:",self.seats)
    
ob = Car("Blue",4)#object creation
ob.col #"Blue"
ob.seats #4
ob.show()